### ISSUE TITLE
Add F0.5 score to metrics

### ISSUE THREAD

**Issue by AlexanderGeiger** [sintel-dev/Orion#40]
### Description

In the original paper a f0.5 score is used for evaluating the result. We might want to add that metric as well.
sklearn.metrics.fbeta_score with beta=0.5 should achieve that.
